src/cljs/athens/devcards/app_toolbar.cljs

```diff
@@ -193,8 +202,13 @@
             :content [:div (use-style modal-contents-style)
                       ;; TODO: Write intro copy
                       [:p "Some helpful framing about what Athens does and what users should expect. Athens is not Roam."]
+                      [:p "To export a Roam database to the " [:code "json triplet"] " format that Athens understands, refer to the official Athens documentation."]
```

Comment:
Maybe we need more docs regarding different formats to import/export? Since we don't currently support Roam's official export functionality